### DISCUSSION TITLE
label_values function for prometheus can use "/api/v1/label/label/values" instead of "/api/v1/series" API ?

### DISCUSSION THREAD

**Discussion by chenlujjj** [grafana/grafana#43001] [Observability]
In some of our use cases, `label_values(metric, label)` function returns response very slowly (more than 10 seconds). We find out that it calls prometheus `/api/v1/series API` and the response body size is about 150MB. 

I'm wondering if grafana can use `/api/v1/label/label/values` instead since it is faster and response body is much smaller? Prometheus has provided `match[]=<series_selector>` parameter for this API since version 2.24.

**Comment by aocenas** [comment]
There is already a feature request for that https://github.com/grafana/grafana/issues/33487

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
